jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt

```diff
@@ -98,13 +99,15 @@ class SetupAuthenticationDialog(
     private val scopes: List<String> = emptyList(),
     private val state: SetupAuthenticationDialogState = SetupAuthenticationDialogState(),
     private val tabSettings: Map<SetupAuthenticationTabs, AuthenticationTabSettings> = emptyMap(),
-    private val promptForIdcPermissionSet: Boolean = false,
+    private val promptForIdcPermissionSet: Boolean = true,
```

Comment:
let's keep this as false since there are more cases where we don't need account/permissions

Reply:
sure